docs/installation/index.md

```diff
@@ -0,0 +1,84 @@
+# Installation
+
+## Step 1: **Requirements**
+
+In this section we will cover some of the key dependencies for all of the deployment modes.
+
+-  [Docker](https://docs.docker.com/get-docker/) - For building and running containers
+-  [Docker Compose](https://docs.docker.com/compose/install/) - For managing multiple containers
+-  [Python 3.12](https://www.python.org/downloads/) - For intellisense and linting (not explicitly needed to run the project due to docker, but recommended for development)
+-  [pip](https://pip.pypa.io/en/stable/installation/) - For installing poetry
+-  [poetry](https://python-poetry.org/docs/) - For managing python packages
+-  [Make](https://www.gnu.org/software/make/) - For running commands in the `Makefile`
+-  [Poppler](https://poppler.freedesktop.org/) - For Document Ingestion and OCR
+-  [Tesseract](https://github.com/tesseract-ocr/tesseract) - For Document Ingestion and OCR
+
+
+### Brew
+
+If you're on a Mac, you can install the above dependencies using [brew](https://brew.sh/). 
+
+```bash
+brew install docker docker-compose python@3.12 make poppler tesseract
+```
+
+### Ubuntu/Debian
+
+If you're on Ubuntu/Debian, you can install the above dependencies using `apt`.
+
+```bash
+sudo apt update
+sudo apt install docker docker-compose python3.12 make poppler-utils tesseract-ocr
+```
+
+If you don't have this python version, we'd recommend using [pyenv](https://github.com/pyenv/pyenv) to manage your python versions.
+
+## Step 2: **Installation and Build**
+
+We'll need to clone the repository and install the dependencies.
+
+```bash
+git clone git@github.com:i-dot-ai/redbox.git
+cd redbox
+poetry install
+```
+
+This may take a while as it installs all the dependencies on a first run.
+
+To build the project, you can use the following command:
+
+```bash
+make build
```

Comment:
Is it worth mentioning that a bare `make` lists all the helpful targets?

Reply:
Great shout, have added!
![Screenshot 2024-07-15 at 11 32 43](https://github.com/user-attachments/assets/81f150c2-fc1d-4c82-a508-4c1b08f36750)